data/decks/vega.json

```diff
@@ -0,0 +1,209 @@
+{
+	"id": "vega",
+	"season": 3,
+	"display_name": "Vega",
+	"cardback": "cardback_defaultexceed.png",
+	"exceed_cost": 3,
+	"ability_effects": [
+		{
+			"timing": "set_strike",
+			"condition": "is_not_critical",
+			"effect_type": "gauge_for_effect",
+			"per_gauge_effect": null,
+			"overall_effect": {
+				"effect_type": "critical",
+				"character_effect": true
+			},
+			"gauge_max": 1
+		},
+		{
+			"timing": "set_strike",
+			"condition": "at_edge_of_arena",
+			"effect_type": "add_attack_effect",
+			"added_effect": {
+				"condition": "not_has_vega_ua_guarantee",
```

Comment:
yeah maybe both this and the boost could set character_bonus, then the set_strike effect just checks that (avoids a bit of repetition)

Reply:
okay, yea, i like that better.